lib/drivers/medtronicDriver.js

```diff
@@ -908,7 +1039,7 @@ module.exports = function (config) {
 
       var sessionInfo = {
         deviceTags: ['insulin-pump'],
-        deviceManufacturers: ['Medtronic'],
+        deviceManufacturers: ['CareLink'], //TODO: change to Medtronic once viz is fixed
```

Comment:
Great, seems to work fine on staging. It did make me realize that I wasn't uploading the *current* settings, only previous changes. I've now added it, but it does make for a bit of an awkward handling of time offsets - as there's no index number I don't think I can use TZOUtil - please check commit 94ab260 to see if I've handled it correctly.